### ISSUE TITLE
All GUI gets disabled (gray) sometimes

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#2169]
#### Please, fill all relevant items: ####

- [no] I have read CONTRIBUTING.md

- [no] I have tried with the latest pre-release version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####

v6.5.1

##### Operating system and version: #####

Linux Debian 

##### Expected behavior: #####

GUI stays accessible (except of may be some intermittent subsecond critical secions).

##### Actual behavior: #####

GUI sometimes locks up (all controls disabled) for about a minute.

##### Steps to reproduce the behavior: #####

I don't know specifics. Maybe it happens after I add (or remove) a download. Maybe while adding download. Maybe just sporadically. CPU is busy while I wait for GUI to unblock.

##### Relevant log file output: #####

Nothing special appear on console. Maybe just

    ERROR   1462643721.12 tunnel_community:136   Can't resolve ip address for ...
    ERROR   1462643721.12 tunnel_community:1197  Dropping data packets while EXITing

**Comment by vi** [comment]
Triggered search => gray waiting. Basically requesting anything to do => waiting.

**Comment by vi** [comment]
After restarting the gray waiting time (for example, after starting a search) dropped to a few seconds.

**Comment by devos50** [comment]
Thank you for your report. I suspect this has something to do with wx in combinations with the calls made on the wx thread. I'm not sure whether we can fix this before 6.6 but in 6.7 the GUI will not block when doing operations in Tribler.

**Comment by lfdversluis** [comment]
Found the problem. https://github.com/Tribler/tribler/blob/next/Tribler/community/tunnel/tunnel_community.py#L131 This call is just like the torrent_checker that I refactored blocking, on the main thread (the wx) thread. Makes no sense and has to be done asynchronously, non-blocking.

@whirm assign this to me as it concerns the tunnels which I will target.

**Comment by lfdversluis** [comment]
Is this before or after Vietna.. err I mean 6.7? :D

**Comment by whirm** [comment]
@lfdversluis I would be really nice to have it fixed for 6.6, 6.7 will simply not have this kind of issues but it will be a while until it's ready.
